internal/internal.go

```diff
@@ -172,7 +173,29 @@ func CombinedOutputTimeout(c *exec.Cmd, timeout time.Duration) ([]byte, error) {
 	if err := c.Start(); err != nil {
 		return nil, err
 	}
+
 	err := WaitTimeout(c, timeout)
+
+	if runtime.GOOS == "windows" {
+		var buf bytes.Buffer
```

Comment:
put this logic within the if-statement in it's own function and unit test and put a comment above the function explaining what it's doing.